py4DSTEM/braggvectors/diskdetection_aiml.py

```diff
@@ -840,7 +837,8 @@ def _integrate_disks(DP, maxima_x, maxima_y, maxima_int, int_window_radius=1):
         disks.append(np.average(disk))
     try:
         disks = disks / max(disks)
-    except:
+    # TODO work out what exception would go here
```

Comment:
TODO

Reply:
I'd guess this is a `ZeroDivisionError`, but I didn't want to guess I can remove the TODO if prefered.